packages/babel-helpers/src/helpers/importDeferProxy.js

```diff
@@ -0,0 +1,31 @@
+/* @minVersion 7.22.0 */
+export default function _importDeferProxy(init) {
```

Comment:
It may be worthy of an assumption to get rid of this helper, since proxy is difficult to optimize and the helper will kick in when people are merely assigning an alias to the namespace object like `const e = ns;`. In this case they are not asking for any reflection support.

Reply:
Would you expect `const e = ns` to trigger the evaluation of the module, under that option? Or should we track usages of `e` and transform them in calls that evaluate the module?

I think this helper will already be used very rarely, because most of the usages of "deferred namesapces" will just be people rewriting `import { val } from "./x"; doSomethingWith(val)` to `import defer * as x from "./x"; doSomethingWith(x.val)`

Reply:
> Or should we track usages of e and transform them in calls that evaluate the module?

I agree with this, but it could be in a future PR.

Reply:
Right, which means we will need to track usages of `e` because we will need to replace `e.something` with `e().something`.